espnet2/train/batch_sampler.py

```diff
@@ -37,22 +34,20 @@ def build_batch_sampler(
         shape_files: Text files describing the length and dimension
             of each features. e.g. uttA 1330,80
         max_lengths: Used for "seq" mode
-        shuffle: If False, the batch are sorted by ascending order.
```

Comment:
The options of shuffle are removed from sampler and instead of it,  the iterator factory class shuffles the samples